packages/ecc-utils-design/demo/code/index.html

```diff
@@ -15,8 +15,31 @@
       import { html, render } from "lit";
       import "../../dist/components/code/index.js";
 
+      const code = `doe: "a deer, a female deer"
+ray: "a drop of golden sun"
+pi: 3.14159
+xmas: true
+french-hens: 3
+calling-birds:
+  - huey
+  - dewey
+  - louie
+  fred
+xmas-fifth-day:
+  calling-birds: four
+  french-hens: 3
+  golden-rings: 5
+  partridges:
+    count: 1
+    location: "a pear tree"
+  turtle-doves: two
+      `;
+
       render(
-        html`<ecc-utils-design-code> </ecc-utils-design-code> `,
+        html`<ecc-utils-design-code
+    language="yaml"
+    code=${code}
```

Comment:
this will need to be changed to `value`, since it was changed in the code component

Reply:
Thanks for the catch